Metadata/MetadataConverter.php

```diff
@@ -63,7 +63,22 @@ public function convert(ClassHierarchyMetadata $metadata)
             if (null !== $classMetadata->arguments) {
                 $definition->setArguments($classMetadata->arguments);
             }
+            if (null !== $classMetadata->autowire) {
+                if (!method_exists($definition, 'setAutowire')) {
+                    throw new InvalidAnnotationException(
+                        sprintf(
+                            "autowiring require symfony >=2.8 on class %s"
```

Comment:
I would reword this a bit in fact ``You must use symfony 2.8 or higher to use autowiring on the class %s.`` what do you think?